main/test/flix/Test.Exp.Force.flix

```diff
@@ -160,6 +160,12 @@ namespace Test/Exp/Force {
         let x = lazy None;
         None: Poly[Unit, Unit] == force x
 
+    @test
+    def testPrecedenceForce01(): Bool =
+        let x = lazy 2;
+        let y = force x + 1;
+        y == 3 and force x == 2
+
```

Comment:
Could I ask for a few more tests, say 3-5-10. They can be very similar. Perhaps with operators of different precedence.

Reply:
What do you think about merging the lazy/force test files? the force test file sort of have to test lazy anyway